try {
  writeMyFile(theData); //Isso pode lan√ßar um erro
} catch (e) {
  handleError(e); // Se temos um erro temos que lidar com ele
} finally {
  closeMyFile(); // Sempre feche o recurso
}

throw "bogus";

console.log(e.message);

